<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlacWindow</class>
 <widget class="QMainWindow" name="PlacWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>176</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="nlabel">
      <property name="text">
       <string>Do zapłaty:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="DoZaplaty">
      <property name="text">
       <string>0.00</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="nabel_3">
      <property name="text">
       <string>1 gr.</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="nlabel_4">
      <property name="text">
       <string>2 gr.</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="nlabel_5">
      <property name="text">
       <string>5 gr.</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="nlabel_6">
      <property name="text">
       <string>10 gr.</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="nabel_7">
      <property name="text">
       <string>20 gr.</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="nlabel_8">
      <property name="text">
       <string>50 gr.</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="nlabel_9">
      <property name="text">
       <string>1 zł.</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="nlabel_10">
      <property name="text">
       <string>2 zł.</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QLabel" name="nlabel_11">
      <property name="text">
       <string>5 zł.</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QSpinBox" name="ngr_1"/>
    </item>
    <item row="4" column="1">
     <widget class="QSpinBox" name="ngr_2"/>
    </item>
    <item row="5" column="1">
     <widget class="QSpinBox" name="ngr_5"/>
    </item>
    <item row="6" column="1">
     <widget class="QSpinBox" name="ngr_10"/>
    </item>
    <item row="7" column="1">
     <widget class="QSpinBox" name="ngr_20"/>
    </item>
    <item row="8" column="1">
     <widget class="QSpinBox" name="ngr_50"/>
    </item>
    <item row="9" column="1">
     <widget class="QSpinBox" name="nzl_1"/>
    </item>
    <item row="10" column="1">
     <widget class="QSpinBox" name="nzl_2"/>
    </item>
    <item row="11" column="1">
     <widget class="QSpinBox" name="nzl_5"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="nlabel_2">
      <property name="text">
       <string>Monety:</string>
      </property>
     </widget>
    </item>
    <item row="12" column="0">
     <widget class="QPushButton" name="Zaplac">
      <property name="text">
       <string>Zaplac</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="twojaSuma">
      <property name="text">
       <string>0.00</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
